ql/experimental/math/tcopulapolicy.cpp

```diff
@@ -73,17 +72,14 @@ namespace QuantLib {
             "Incompatible sample and latent model sizes");
     #endif
 
-        using namespace ext::placeholders;
-
         std::vector<Real> result(probs.size());
         Size indexSystemic = 0;
         std::transform(probs.begin(), probs.begin() + varianceFactors_.size()-1,
-            result.begin(), 
-            ext::bind(&TCopulaPolicy::inverseCumulativeDensity, 
-                                this, _1, indexSystemic++));
```

Comment:
@japari I came across this while converting the bind into a lambda — was this supposed to increment `indexSystemic` at each function call? Because it's only incremented once, when the bind (or the lambda) is created...